docs/develop/03_workshop/04_attester/03_ctype.md

```diff
@@ -50,8 +50,6 @@ Let's have a look at these attributes.
 | `schema` > `$schema`    | The JSON schema on which the CType is based.                             |
 | `schema` > `title`      | The title of the CType.                                                  |
 | `schema` > `properties` | The properties that a claim of type in `$schema` should have.            |
-| `owner`                 | The DID of the user who created this CType.                              |
-| `hash`                  | Most important attribute, the hash is the CType's **digital footprint**. |
```

Comment:
I suggest we move the "digital footprint" idea into the description of the $id prop